ynput/ayon-core
# Clockify: Move client code to server

## Changelog Description
Moved clockify addon logic next to server codebase. This is one more step towards splitting addons to their repos.

## Additional info
From now on Cloclify addon lives out of ayon-core codebase.

## Testing notes:
1. Create clockify package (`python ./server_addons/create_ayon_addons.py --addon clockify`).
2. Create core package too.
3. Upload both packages to server.
4. Use boths addon in bundle.
5. Clockify functionality should be available when running AYON launcher.
